Formula/n/nextflow.rb

```diff
@@ -0,0 +1,49 @@
+class Nextflow < Formula
+  desc "Reproducible scientific workflows"
+  homepage "https://nextflow.io"
+  url "https://github.com/nextflow-io/nextflow/archive/refs/tags/v25.04.6.tar.gz"
+  sha256 "e0891da76f2e17336eaeadac14971a7e69cdc07722d44b605489c4dc6718d6a5"
+  license "Apache-2.0"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
Why is this needed?

Reply:
otherwise it would pick `25.07.0-edge`, yeah, we can update it to do regex match rather than checking the latest